charts/index.yaml

```diff
@@ -0,0 +1,71 @@
+apiVersion: v1
+entries:
+  secrets-store-csi-driver:
+  - apiVersion: v1
```

Comment:
I don't think we need to package all versions of the charts. As we are starting to provide packages with current existing chart, just `0.0.9` should be sufficient.

Reply:
Sounds fair. Removed previous.